src/citrine/resources/project.py

```diff
@@ -223,6 +224,49 @@ def make_private(self,
         })
         return True
 
+    def list_members(self) -> List[User]:
```

Comment:
Is this from develop?

Reply:
yah this includes a develop merge as well.